src/connection_manager.rs

```diff
@@ -80,7 +80,11 @@ where
     type Error = ConnectionError;
 
     async fn connect(&self) -> Result<Self::Connection, Self::Error> {
-        self.run_blocking(|m| m.connect())
+        self.run_blocking(|m| {
+            let conn = m.connect();
+            async_bb8_diesel_new_connection!(|| ());
+            conn
+        })
             .await
             .map(Connection::new)
             .map_err(ConnectionError::Checkout)
```

Comment:
Gah. It's too far below in this file to let me comment, but we should probably fire the `async_bb8_diesel_query` probe(s) around line 120ish of this file too.

This crate tries to give some flexibility over how queries are issued:
1. They can check out a connection and issue queries to it (like you instrumented in `connection.rs`)
2. Or they can just issue queries to the "pool" object directly (which gets sent to `ConnectionManager`). This lets the pool sort out the details of "what connection to use".

But a byproduct is that there are two spots where we're potentially issuing `batch_execute` on a connection.